api/db/populate-attendants.js

```diff
@@ -0,0 +1,67 @@
+const { getRandomSubset } = require('./utils');
+
+const User = require('../models/User');
+const Event = require('../models/Event');
+const Attendant = require('../models/Attendant');
+
+let displayLogs;
+
+const createAttendant = async (attendantParams) => {
+  const attendant = new Attendant({
+    event: attendantParams.event,
+    user: attendantParams.user,
+  });
+  await attendant.save();
+
+  if (displayLogs) {
+    console.log('\n', '\x1b[0m', `New attendant created: ${attendant}`);
+  }
+};
+
+const createAttendants = async (verbose) => {
+  console.log('\n', '\x1b[0m', 'Populating attendants collection...');
+
+  let usersList = await User.distinct('_id');
+  if (usersList.length === 0) {
+    console.log(
+      '\x1b[33m',
+      'The users collection does not exist. Aborted attendants collection population.'
+    );
+    return;
+  }
+
+  let eventsList = await Event.find();
+  if (eventsList.length === 0) {
+    console.log(
+      '\x1b[33m',
+      'The events collection does not exist. Aborted attendants collection population.'
+    );
+    return;
+  }
+
+  displayLogs = verbose;
+  let attendantsList = await Attendant.find();
+  const attendantsArray = [];
+
+  for (const event of eventsList) {
```

Comment:
In other places, I've been using the `forEach()` method instead. Maybe we should change it here too for consistency, but I don't feel too strongly about that.

Reply:
The only thing with `forEach` is that it cant be asynchronous (last time I checked). That's why I went with the `for..of` approach.